src/Ledger/Utxo.lagda

```diff
@@ -320,14 +322,14 @@ isAdaOnlyᵇ v = toBool (policies v ≡ᵉ coinPolicies)
 \end{code}
 \begin{code}
 
-feesOK : PParams → Tx → UTxO → Bool
-feesOK pp tx utxo = minfee pp utxo tx ≤ᵇ txfee
-                  ∧ not (≟-∅ᵇ (txrdmrs ˢ))
-                  =>ᵇ ( allᵇ (λ (addr , _) → ¿ isVKeyAddr addr ¿) collateralRange
-                      ∧ isAdaOnlyᵇ bal
-                      ∧ (coin bal * 100) ≥ᵇ (txfee * pp .collateralPercentage)
-                      ∧ not (≟-∅ᵇ collateral)
-                      )
+feesOK : (pp : PParams) → Tx → UTxO → Bool
```

Comment:
This shouldn't be dependently quantified, since it doesn't do anything and we show it in some versions of the PDF.